### PR TITLE
Improve probe helper logging

### PR DESCRIPTION
**Pull request by tommyreddad**
Improve probe helper logging

## Proposed Changes

- Enable debug-level logging. Otherwise, all debug level log messages are lost.
- Use info-level logs for specific probe forward/receive event success.
- Use debug-level logs in generic event handling to reduce log clutter.

### CODE REVIEW

File: test/test_images/probe_helper/probe/handlers/cloudscheduler.go [hunk 1]
```diff
@@ -102,6 +102,7 @@ func (p *CloudSchedulerSourceProbe) Receive(ctx context.Context, event cloudeven
 
 	timestampID := fmt.Sprint(event.Extensions()[utils.ProbeEventReceiverPathExtension])
 	p.EventTimes.Times[timestampID] = time.Now()
+	logging.FromContext(ctx).Info("Successfully received CloudScheduleSource probe event")
```
Comment (stuart-xyz) [inline test/test_images/probe_helper/probe/handlers/cloudscheduler.go:105]: Nit: typo for CloudSchedulerSource

File: test/test_images/probe_helper/probe/handlers/ping.go [hunk 2]
```diff
@@ -70,6 +70,7 @@ func (p *PingSourceProbe) Forward(ctx context.Context, event cloudevents.Event)
 	p.EventTimes.RLock()
 	defer p.EventTimes.RUnlock()
 
+	logging.FromContext(ctx).Infow("Checking last observed PingSource tick")
```
Comment (stuart-xyz) [inline test/test_images/probe_helper/probe/handlers/ping.go:73]: Nit: seems like this could also just be `logging.FromContext(ctx).Info(...)`?

Comment (tommyreddad) [reply test/test_images/probe_helper/probe/handlers/ping.go]: Done.

File: test/test_images/probe_helper/probe/handlers/cloudscheduler.go (hunk 1 above)
Comment (tommyreddad) [reply test/test_images/probe_helper/probe/handlers/cloudscheduler.go]: Done.

**Pull request merged**